bindings/pyroot_experimental/PyROOT/python/ROOT/pythonization/_ndarray.py

```diff
@@ -0,0 +1,38 @@
+# Author: Stefan Wunsch CERN, Vincenzo Eduardo Padulano (UniMiB, CERN) 07/2019
```

Comment:
Since this is not strictly a pythonisation of `_ndarray`, can we call the file `_rdf_helpers.py` ?

Reply:
sure!

Reply:
Maybe `_rdf_utils.py`, mimicking the already used `RDFUtils` in C++ rdf ?

Reply:
:+1: